src/api/history/index.ts

```diff
@@ -1,13 +1,18 @@
-
-import { expectResponseCode, getApiUrl, defaultFetchConfig } from '../utils'
 import { HistoryEntry } from '../../components/history-page/history-page'
+import { defaultFetchConfig, expectResponseCode, getApiUrl } from '../utils'
 
 export const getHistory = async (): Promise<HistoryEntry[]> => {
   const response = await fetch(getApiUrl() + '/history')
   expectResponseCode(response)
   return await response.json() as Promise<HistoryEntry[]>
 }
 
+export const getHistoryEntry = async (noteId: HistoryEntry['id']): Promise<HistoryEntry> => {
+  const response = await fetch(getApiUrl() + '/history' + noteId)
```

Comment:
This looks as if a slash is missing or does the noteId start with one?

Reply:
nope this was a mistake

Reply:
done